File: Patterns/7. Pyramid Patterns/68. Pyramid Alphabets/pa.java
import java.util.*;
public class pa{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            // print spaces
            for(int j=1; j<=n-i; j++){
                System.out.print("  ");
            }
            // print first half
            for(int j=1; j<=i; j++){
                System.out.print((char)('A'+j-1) + " ");
            }
            // print second half
            for(int j=i-1; j>=1; j--){
                System.out.print((char)('A'+j-1) + " ");
            }
            System.out.println();
        }
    }
}

/*
Output

        A 
      A B A 
    A B C B A 
  A B C D C B A 
A B C D E D C B A 

*/

File: Patterns/07. Pyramid Patterns/68. Pyramid Alphabets/pa.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    # print spaces
    for j in range(1,n-i+1):
        print(end='  ')
    # print first half
    for j in range(1,i+1):
        print(chr(ord('A')+j-1),end=' ')
    # print second half
    for j in range(i-1,0,-1):
        print(chr(ord('A')+j-1),end=' ')
    print()


'''

Output:

        A 
      A B A 
    A B C B A 
  A B C D C B A 
A B C D E D C B A 

'''